-added time down count

Christy233 opened the pull request.

Christy233 committed:
-added time down count

liaohenry2310 commented:
The TimeDown script looks really good. I would suggest three things:

1. Add pausing functionality to the TimeDown script, so we can control when the timer can start counting down. Perhaps add a public function called Pause() and Unpause() so other gameobjects can interface with it. It would probably be best if the TimeDown script defaults to a paused state.

2. Create a prefab called something like "Timer" that has the TimeDown script attached to it. This is just so other people can use your TimeDown script easily in their own Unity scenes.

3. Do remember to create and work on your own test scenes to avoid possible merge conflicts with other people.

CyroPCJr committed:
CodeReview Cyro

- Made some test with different time float;
- Good practice to keep name variables more reliable for better understand;
- Changed the font size to 30 pixel for test.
